ibm/mas_devops/roles/gencfg_watsonstudio/README.md

```diff
@@ -0,0 +1,60 @@
+suite_config
```

Comment:
New roles need to be integrated into the documentation ... see copy role docs script and mkdocs.yml